.github/workflows/automerge.yml

```
@@ -18,7 +18,7 @@ jobs:
           - x64
     steps:
       - uses: actions/checkout@af513c7a016048ae468971c52ed77d9562c7c819 # v1.0.0
-      - uses: julia-actions/setup-julia@4469f5b9da960ff174efa528fea5889f8e873609 # v1.2.2
+      - uses: julia-actions/setup-julia@d26d1111976eae5f00db04f0515ab744ec9cd79e # v1.3.1
```

I took that hash from the [README](https://github.com/julia-actions/setup-julia/blob/405bfcbfbd71503af238843c921b4a623e815267/README.md#versioning)...
```suggestion
      - uses: julia-actions/setup-julia@3dcc911681c8af6e64fdc847e77c5e98f9d1611e # v1.3.1
```

---

So I have no idea where I got `d26d1111976eae5f00db04f0515ab744ec9cd79e` from, it doesn't appear to be a valid commit.